code/clue.js

```diff
@@ -1,59 +1,171 @@
+  lastName: 'Plum',
+  description: 'A professor in history and criminal technology.',
+  age: '62 years old',
+  image: 'assets/plum.png',
+  occupation: 'Detective'
+}
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  description: 'A famous actor, been in many movies.',
+  age: '47 years old',
+  image: 'assets/scarlet.png',
+  occupation: 'Actor'
+}
+const mrsPeacock = {
+  firstName: 'Eleonor',
+  lastName: 'Peacock',
+  description: 'Degree in criminal history and writing. Love tea and cookies.',
+  age: '59 years old',
+  image: 'assets/peacock.png',
+  occupation: 'Author'
+}
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  description: 'Passion of the military and hunting on his free time.',
+  age: '76 years old',
+  image: 'assets/mustard.png',
+  occupation: 'Military Officer'
+}
+const mrsWhite = {
+  firstName: 'Jane',
+  lastName: 'White',
+  description: 'A nurse that worked as a midwife all her life.',
+  age: '74 years old',
+  image: 'assets/white.png',
+  occupation: 'Nurse'
+}
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
-
+//OBJECTS FOR ALL THE WEAPONS
 const rope = {
-  name: 'rope',
+  name: 'Rope',
   weight: 10
 }
+const knife = {
+  name: 'Knife',
+  weight: 1
+}
+const candlestick = {
+  name: 'Candlestick',
+  weight: 0.1
+}
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 5
+}
+const poison = {
+  name: 'Poison',
+  weight: 0.2
+}
+const axe = {
+  name: 'Axe',
+  weight: 6
+}
+const bat = {
+  name: 'Bat',
+  weight: 1.5
+}
+const trophy = {
+  name: 'Trophy',
+  weight: 4
+}
+const pistol = {
+  name: 'Pistol',
+  weight: 2
+}
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
-
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
-
+//ARRAYS FOR ALL THE SUSPECTS, WEAPONS & ROOMS
 const suspects = [
   mrGreen,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = [
+  'Dining Room', 
+  'Conservatory', 
+  'Kitchen', 
+  'Study', 
+  'Library', 
+  'Billiard Room', 
+  'Lounge', 
+  'Ballroom', 
+  'Hall', 
+  'Spa', 
+  'Living Room', 
+  'Observatory', 
+  'Theatre', 
+  'Guest House', 
+  'Patio'
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
 
-// This function will be invoked when you click on the killer card.
+//OBJECT THAT KEEPS THE MYSTERY.
+const mystery = {
+killer: '',
+weapon: '',
+room: ''
+}
```

Comment:
Let's prefer `null` values instead of `''`

Reply:
Fixed :)